rdrf/rdrf/settings.py

```diff
@@ -315,11 +315,23 @@
 # NB. This initialises the email notification form
 DEFAULT_FROM_EMAIL = env.get('default_from_email', 'No Reply <no-reply@mg.ccgapps.com.au>')
 SERVER_EMAIL = env.get('server_email', DEFAULT_FROM_EMAIL)
-EMAIL_BACKEND = 'anymail.backends.mailgun.EmailBackend'
-
-ANYMAIL = {
-    'MAILGUN_API_KEY': env.get('DJANGO_MAILGUN_API_KEY', ''),
-}
+EMAIL_BACKEND = env.get('EMAIL_BACKEND', 'anymail.backends.mailgun.EmailBackend')
+
+if EMAIL_BACKEND == 'smtp':
```

Comment:
Indicate somewhere in the ticket for this that this is referring to another ticker